README.md

```diff
@@ -1,26 +1,151 @@
 # Stencil Icon
 
-Stencil Icon Component
+A container for displaying SVG icons from a `<symbol>`-based SVG sprite.
 
-[Link to demo](#)
+**View demo (coming soon)**
 
 ## Requirements
 
-- Sass
-- AMD loader
-- Dust templating
+- AdaptiveJS 2.1 or greater
 
 ## Installation
 
-Installation instructions. Be as brief as possible without leaving out necessary requirements.
+```shell
+cd my/adaptive/project
+grunt component:install:icon
+```
 
 ## Usage
 
-- How to
+The Icon component ships with a default icon set, [Shoppicon](http://github.com/ry5n/shoppicon). Usage depends in whether you want to use the default set, or a custom set.
 
-## Development
+### Default icon set
 
-* run `npm install`
-* run `bower install`
-* run `grunt serve`
-* navigate to [localhost:3000/tests/visual](http://localhost:3000/tests/visual)
+1. In your adaptive project, open `app/global/base-view.js`, and add the following dependency:
+
+    ```javascript
+    define(['$', ..., 'text!bower_components/shoppicon/svg-sprite/shoppicon.svg'], function($, ..., icons) {...});
+    ```
+
+    Then, within the views’s returned context, add a key for the icon sprite:
+
+    ```javascript
+    return {
+        preProcess: function(context) {...},
+        postProcess: function(context) {...},
+        context: {
+            ...
+            icons: icons,
+        },
+    };
+    ```
+
+    Finally, within `app/global/base.dust`, include the svg sprite *at the very top of the body tag*, like so:
+
+    ```html
+    {body|openTag|s}
+        <div hidden>{icons|s}</div>
+        
+        {+bodyBlock}
+        ...
+        {/bodyBlock}
+    </body>
+    ```
+
+2. Use the Icon component’s dust helper to render icons. For example, to show a hamburger icon:
+
+    ```html
+    {@c-icon set="shoppicon" name="menu" /}
```

Comment:
In regards to the `set` attribute problem, is it possible to set this on `{icons|s}` somehow?

```html
{icons set="shoppicon"|s}

<!-- elsewhere -->
{@c-icon name="menu" /}
```